andela/ah-the-unsullied-frontend
# #162949176 Enable Article creation

#### What does this PR do?

Enable User to create an article

#### Description of Task to be completed?
- [x] User can create an article

#### How should this be manually tested?

1. Clone the repository
```
git clone https://github.com/andela/ah-the-unsullied-frontend.git
```
2. Cd into the cloned repo
```
cd ah-the-unsullied-frontend
```
3. Checkout to this branch
```
git checkout ft-create-article-162949176
```
4. Install the dependencies of the project
```
npm install
```
5. Run the app
```
npm start
```
6. Visit the default route `http://localhost:3000/` to see a create article link

7. To run tests, run `npm test`

#### Any background context you want to provide?
You need to login to create an article

#### What are the relevant pivotal tracker stories?
[#162949176 ](https://www.pivotaltracker.com/story/show/162949176)

#### Screenshots (if appropriate)

#### Questions:
